### DISCUSSION TITLE
!ERROR! | |Attempting to update the LXC container with "Pingvin Share LXC"

### DISCUSSION THREAD

**Discussion by ScienceWolf666** [tteck/Proxmox#1903] [Help Needed]
Typed into the LXC container terminal ( Pingvin Share LXC ) script [ 
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/pingvin.sh)" ] to update the container 

LXC Terminal Output LOG:

 ✓ Stopped Pingvin Share
 - Updating Pingvin Share...
[ERROR] in line 63: exit code 1: while executing command git checkout $(git describe --tags `git rev-list --tags --max-count=1`) &> /dev/null

root@PingvinShare-LXC:/# 

//END_OF_LOG

umm heh ^^ plsss help ;3

**Comment by tteck** [comment]
Running the command below should get your Pingvin Share to a functional state while I install it and investigate further.

```
pm2 start pingvin-share-backend pingvin-share-frontend
```

**Comment by tteck** [comment]
Can't find any issues.
```
    ____  _                   _          _____ __                  
   / __ \(_)___  ____ __   __(_)___     / ___// /_  ____ _________
  / /_/ / / __ \/ __ `/ | / / / __ \    \__ \/ __ \/ __ `/ ___/ _ \
 / ____/ / / / / /_/ /| |/ / / / / /   ___/ / / / / /_/ / /  /  __/
/_/   /_/_/ /_/\__, / |___/_/_/ /_/   /____/_/ /_/\__,_/_/   \___/
              /____/
 ✓ Stopped Pingvin Share
 ✓ Updated Pingvin Share
 ✓ Started Pingvin Share
 ✓ Updated Successfully
root@pingvin:~# 
```
You may have to run each command one at a time without the `&>/dev/null` to see the exact issue.

```
pm2 stop pingvin-share-backend pingvin-share-frontend
cd /opt/pingvin-share
git fetch --tags
git checkout $(git describe --tags `git rev-list --tags --max-count=1`)
cd backend
npm run build
cd ../frontend
npm run build
pm2 start pingvin-share-backend pingvin-share-frontend
```
The update process is from their Github page (https://github.com/stonith404/pingvin-share#upgrade-to-a-new-version)

**Reply by mundeczek** [reply]
@tteck 
update process has been improved https://github.com/stonith404/pingvin-share/blob/main/README.md and https://github.com/stonith404/pingvin-share/issues/424
When I try to update with your script I get:
```
[ERROR] in line 69: exit code 0: while executing command npm run build &> /dev/null
```
update step by step according to their Github page is successful

**Comment by Rems-38600** [comment]
Bonjour,
J'ai mise à jour la version 1.0.2 vers la version 1.1.3 en faisant la manipulation suivante:
cd /opt/pingvin-share
Puis
git reset --hard
et enfin la commande update a fonctionner
 ✓ Stopped Pingvin Share
 ✓ Updated Pingvin Share
 ✓ Started Pingvin Share
 ✓ Updated Successfully
root@pingvin:/opt/pingvin-share# 

Good morning,
I updated version 1.0.2 to version 1.1.3 by doing the following:
cd /opt/pingvin-share
Then
git reset --hard
and finally
update
The command worked
 ✓ Stopped Pingvin Share
 ✓ Updated Pingvin Share
 ✓ Started Pingvin Share
 ✓ Updated Successfully
root@pingvin:/opt/pingvin-share#
